workspaces/src/worker/impls.rs

```diff
@@ -127,6 +133,10 @@ where
         self.client().view_state(contract_id.clone(), prefix).await
     }
 
+    pub async fn view_latest_block(&self) -> anyhow::Result<Block> {
+        self.client().view_block(None).await.map(Into::into)
+    }
```

Comment:
Hmm, does this mean there will be no way to view an arbitrary (non-last) block?

Reply:
yup, and because it would only be available for archival networks. The archival stuff in workspaces is very limited to just working with importing contracts

Reply:
Ah, ok, I was confused because the underlying client function you implemented supports any kind of block.